packages/experimental/Text/src/Variants.ts

```diff
@@ -0,0 +1,71 @@
+import { Variants } from '@fluentui-react-native/theme-types';
+import { Text } from './Text';
+import { createFontAliasTokens } from '@fluentui-react-native/win32-theme';
+
+const variantsWin32: Partial<Variants> = createFontAliasTokens('light');
+
+export const Caption1 = Text.customize({
+  fontFamily: variantsWin32.caption1.face,
```

Comment:
>fontFamily [](http://example.com/codeflow?start=2&length=10)

I wonder if it would be simpler to use variant here instead of specifying fontFamily/fontSize/fontWeight separately?

Reply:
Yeah, eventually we do want to do that. When we deprecate old variants and support v2 variant alias tokens for all platforms, we should define define V2 variants
 in defaultTypography and use the variant token to customize here. Right now, the issue is that I cannot define default token values for just win32 without causing an error on the other platforms I just made a note of that in this file as well as a comment.